ingest/downloader/flattener.py

```diff
@@ -121,3 +137,7 @@ def _get_concrete_entity(self, content: dict):
     def _is_project(self, parent_key: str):
         entity_type = parent_key.split('.')[0]
         return entity_type == 'project'
+
+
+class Error(Exception):
```

Comment:
If you create custom exceptions, please name them in a way that the reader of stacktraces or logs would recognise and understand them.

Reply:
Yup will apply in the future. For now, I've just reused ValueError for all errors here as suggested.